src/redux/actions/countries.js

```diff
@@ -0,0 +1,5 @@
+export const GET_COUNTRIES = 'covid_data/countries/GET_COUNTRIES';
+
+const getCountriesAction = (payload) => ({ type: GET_COUNTRIES, payload });
+
+export default getCountriesAction;
```

Comment:
- **[OPTIONAL]** It is good to try other ways to use redux 👏🏽 👏🏽 👏🏽, but we could use the ducks pattern to avoid creating folders. It makes more sense for these pieces `{actionTypes, actions, reducer}` to be bundled together in an isolated module that is self-contained and can even be packaged easily into a library.